service/component-samples/manufacturer/src/main/java/org/fido/iot/sample/ManufacturerContextListener.java

```diff
@@ -162,7 +162,7 @@ protected void dispatching(Composite request) {
       protected void failed(Exception e) {
         StringWriter writer = new StringWriter();
         try (PrintWriter pw = new PrintWriter(writer)) {
-          e.printStackTrace(pw);
+          e.getMessage();
```

Comment:
Should we be logging these as well using SOP?

Reply:
Agree, fixed.